gegelatilib/include/importer/tpgGraphDotImporter.h

```diff
@@ -0,0 +1,400 @@
+		*	I[0-9]+       looks for a I followed by a number. the number will be stored in a group
+		*	.*			  the following can be any sequence of character
+		*
+		*   Example:
+		*	P22 -> I22[style=invis]				Should pass
+		*	T0 -> P22 -> A11					Should not pass
+		*/
+		const std::string linkProgramInstructionRegex;
+
+		/**
+		*	\brief contains the regex to identify a Team -> Program -> Action Link
+		*
+		*	this regex values "T([0-9]+)\\x20->\\x20P([0-9]+)\\x20->\\x20A([0-9]+).*"
+		*
+		*	Explanation :
+		*
+		*	T[0-9]+       looks for a T followed by a number. the number will be stored in a group
+		*	\\x20	      looks for a whitespace 
+		*	->			  looxs for the sequence '->'
+		*	\\x20	      looks for a whitespace 
+		*	P[0-9]+       looks for a P followed by a number. the number will be stored in a group
+		*	\\x20	      looks for a whitespace 
+		*	->			  looxs for the sequence '->'
+		*	\\x20	      looks for a whitespace 
+		*	A[0-9]+       looks for a A followed by a number. the number will be stored in a group
+		*	.*			  the following can be any sequence of character
+		*
+		*   Example:
+		*	P22 -> I22[style=invis]				Should not pass
+		*	T0 -> P22 -> A11					Should pass
+		*/
+		const std::string linkProgramActionRegex;
+
+		/**
+		*	\brief contains the regex to identify a Team -> Program -> Team Link
+		*
+		*	this regex values "T([0-9]+)\\x20->\\x20P([0-9]+)\\x20->\\x20T([0-9]+).*"
+		*
+		*	Explanation :
+		*
+		*	T[0-9]+       looks for a T followed by a number. the number will be stored in a group
+		*	\\x20	      looks for a whitespace 
+		*	->			  looxs for the sequence '->'
+		*	\\x20	      looks for a whitespace 
+		*	P[0-9]+       looks for a P followed by a number. the number will be stored in a group
+		*	\\x20	      looks for a whitespace 
+		*	->			  looxs for the sequence '->'
+		*	\\x20	      looks for a whitespace 
+		*	T[0-9]+       looks for a T followed by a number. the number will be stored in a group
+		*	.*			  the following can be any sequence of character
+		*
+		*   Example:
+		*	P22 -> I22[style=invis]				Should not pass
+		*	T0 -> P22 -> T11					Should pass
+		*/
+		const std::string linkProgramTeamRegex;
+
+		/**
+		*	\brief contains the regex to identify a Team -> Program Link
+		*	the outgoing program vertex must already have been linked
+		*
+		*	this regex values "T([0-9]+)\\x20->\\x20P([0-9]+)"
+		*
+		*	Explanation :
+		*
+		*	T[0-9]+       looks for a T followed by a number. the number will be stored in a group
+		*	\\x20	      looks for a whitespace 
+		*	->			  looxs for the sequence '->'
+		*	\\x20	      looks for a whitespace 
+		*	P[0-9]+       looks for a P followed by a number. the number will be stored in a group
+		*
+		*   Example:
+		*	P22 -> I22[style=invis]				Should not pass
+		*	T0 -> P22 -> T11					Should pass
+		*/
+		const std::string addLinkProgramRegex;
+
+
+		/**
+		*	\brief used to pass the group that match from the previously executed regex
```

Comment:
Please list the functions in which this attribute is used.

Why no passing this are arguments to functions instead of being an attribute?